src/main/java/com/choelinska/horrorlibz/controllers/UserController.java

```diff
@@ -0,0 +1,33 @@
+package com.choelinska.horrorlibz.controllers;
+
+import com.choelinska.horrorlibz.model.dto.UserTo;
+
+import com.choelinska.horrorlibz.model.entities.UserEntity;
+import com.choelinska.horrorlibz.services.UserService;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Sort;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/user")
+public class UserController {
+
+    private UserService userService;
+
+    @Autowired
```

Comment:
It's better practice to inject dependencies using the constructor instead of the @Autowired. Please change it to the same way you inject the user service.